src/client/gpm_common.c

```diff
@@ -799,3 +799,17 @@ void gpm_free_xdrs(int proc, union gp_rpc_arg *arg, union gp_rpc_res *res)
     xdr_free(gpm_xdr_set[proc].arg_fn, (char *)arg);
     xdr_free(gpm_xdr_set[proc].res_fn, (char *)res);
 }
+
+bool gpm_equal_oids(gss_const_OID a, gss_const_OID b)
```

Comment:
I recognize you're just moving around existing code, but: can we use `gss_oid_equal()` (in gssapi_ext.h) for this?  It doesn't go through mechglue, if that's the concern.

Reply:
Ok, I'll change it, the function definition seem a matching drop in for our code, even if the signature returns int instead of bool